src/Maths/Silk.NET.Intrinsics/Common/WorkUnit128.cs

```diff
@@ -0,0 +1,11 @@
+using System.Runtime.Intrinsics;
+
+namespace Silk.NET.Intrinsics
+{
+    internal struct WorkUnit128<T> where T:unmanaged
+    {   
+        public WorkUnitFlags Flags { get; set; }
+        public Vector128<T> Vector { get; set; }
+        public unsafe fixed byte Padding[16];
```

Comment:
Why is `fixed byte Padding` and why does it need both the `Vector` and the padding?

Reply:
This is to ensure that you can convert WorkUnit128 to WorkUnit and vice versa using Unsafe.As.

Reply:
Would it be better to have a `Vector128<T> Reserved { get; set; }` or `Vector128<T> Upper` ?

Reply:
I don't know, would it? As long as the space gets filled I suppose it doesn't really matter, the WorkUnitXXX types aren't really public-facing APIs.

Reply:
The fixed sized buffers generally generate security cookies and other bits. Also worse codegen due to being many more fields. Having it be a `Vector128<T>` would remove that and clarify how the bits are reserved and may be interpreted.
It would also allow it to be interpreted as an HVA struct for ABI purposes (assuming you do get rid of `Flags` like you mentioned might be a consideration below).

Reply:
Ah fair enough, the more you know :D 

Will implement.